src/moneypenny/config.py

```diff
@@ -40,3 +40,42 @@ class Configuration:
     Set by the ``DOCKER_SECRET_NAME`` environment variable,
     configured by the helm chart.
     """
+
+    config_dir: str = os.getenv(
+        "MONEYPENNY_CONFIG_DIR", "/opt/lsst/software/moneypenny/config"
+    )
+    """Directory where the Moneypenny ConfigMaps will be mounted.  Leave
+    at default in normal operation.
+    """
+
+    dossier_path: str = os.getenv("DOSSIER_FILE", "/opt/dossier")
+    """Path to the dossier file's mountpoint created in the initContainers.
+    Leave at default in normal operation.  Note that it does not contain the
+    filename (usually "dossier.json")
+    """
+
+    M: str = os.getenv("M_FILE", f"/{config_dir}/M/m.yaml")
+    """Path to M's standing orders.  Leave at default in normal operation.
+    """
+
+    quips: str = os.getenv("QUIP_FILE", f"{config_dir}/quips/quips.txt")
+    """Path to Moneypenny's quip file.  Leave at default in normal operation.
+    """
+
+    moneypenny_timeout: int = int(os.getenv("MONEYPENNY_TIMEOUT") or "300")
+    """Timeout (in seconds) to wait for all containers in the action pod to
+    complete.  Defaults to 300.
+    """
+
+    null_container_image: str = os.getenv(
+        "NULL_CONTAINER_IMAGE", "lsstsqre/farthing:latest"
+    )
+    """Docker image specification for an "exit with return code zero"
+    do-nothing image.  Defaults to "lsstsqre/farthing:latest" but, for
+    instance, "library/alpine:latest" works fine too.
+    """
+
+    # This is not user-configurable; K8s specifies it.
+    namespace_file: str = (
+        "/var/run/secrets/kubernetes.io/serviceaccount/namespace"
+    )
```

Comment:
Nit: If it's not configurable, I would move it into a `moneypenny.constants` module.

Reply:
Actually, since it's not configurable, and nobody cares about what the path is, but rather what the namespace is, you could make this a property that reads the file, or just read the file at __init__ time and store it as a property here.  That would also clean up some of the namespace parts in the kubernetes.py also reading the namespace file

Reply:
Removed; it's just a file-scoped constant in the kubernetes driver now.